prow/config.yaml

```diff
@@ -1727,6 +1727,49 @@ postsubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+    - name: ci-kubernetes-e2e-gce-1-6-1-7-master-skew
+      agent: kubernetes
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/e2e-kubeadm:v20170927-71ecbe7f
+          args:
+          - "--repo=k8s.io/kubernetes=release-1.7"
```

Comment:
Since this is a CI job, don't we want this to be

    - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"

like its parent job, so it actually tests the given commit? Otherwise, we introduce a race condition where a new commit could be merged after the bazel job gets triggered (running against a different commit), and afterward the kubeadm-based job tries to look up the .deb artifacts in a different location because the commit will be different if it just tries to use the newest commit from the branch.

Reply:
Changed it to "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"